includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,226 +1,19 @@
-				],
-			],
-			"links" => [
-				[
-					"type" => "authentication-link",
-					"title" => [
-						"type" => "translatable-text",
-						"key" => "global-navigation-userinfo-signin-title",
-					],
-					"href" => "https://www.wikia.com/signin",
-					"param-name" => "redirect",
-				],
-				[
-					"type" => "authentication-link",
-					"title" => [
-						"type" => "translatable-text",
-						"key" => "global-navigation-userinfo-register-title",
-					],
-					"subtitle" => [
-						"type" => "translatable-text",
-						"key" => "global-navigation-userinfo-register-description",
-					],
-					"href" => "https://www.wikia.com/register",
-					"param-name" => "redirect"
-				],
-			],
-		],
-		"user" => [
-			"header" => [
-				"type" => "avatar",
-				"username" => [
-					"type" => "text",
-					"value" => "Username",
-				],
-				"url" => "http://static.wikia.nocookie.net/591dee3c-0ce9-48c4-96ee-a7bc437c238f/scale-to-width-down/50",
-			],
-			"links" => [
-				[
-					"type" => "link-text",
-					"href" => "#",
-					"title" => [
-						"type" => "translatable-text",
-						"key" => "global-navigation-userinfo-profile",
-					],
-				],
-				[
-					"type" => "link-text",
-					"href" => "#",
-					"title" => [
-						"type" => "translatable-text",
-						"key" => "global-navigation-userinfo-talk",
-					],
-				],
-				[
-					"type" => "link-text",
-					"href" => "#",
-					"title" => [
-						"type" => "translatable-text",
-						"key" => "global-navigation-userinfo-preferences",
-					],
-				],
-				[
-					"type" => "link-text",
-					"href" => "http://community.wikia.com/wiki/Help:Contents",
-					"title" => [
-						"type" => "translatable-text",
-						"key" => "global-navigation-userinfo-help",
-					],
-				],
-				[
-					"type" => "authentication-link",
-					"href" => "http://starwars.wikia.com/wiki/Special:UserLogout",
-					"title" => [
-						"type" => "translatable-text",
-						"key" => "global-navigation-userinfo-signout",
-					],
-					"param-name" => "returnto"
-				],
-			],
-		],
-		"notifications" => [
-			"module" => [
-				"type" => "notifications",
-				"url" => "#",
-				"header" => [
-					"type" => "line-image",
-					"image" => "wds-icons-bell",
-					"title" => [
-						"type" => "translatable-text",
-						"key" => "global-navigation-notifications",
-					],
-				],
-			],
-		],
-		"create_wiki" => [
-			"header" => [
-				"type" => "link-text",
-				"title" => [
-					"type" => "translatable-text",
-					"key" => "wikia-create-wiki-link-start-wikia",
-				],
-				"href" => "http://www.wikia.com/Special:CreateNewWiki",
-			],
-		],
-
-	];
-
 	public function getFooter() {
 		$params = $this->getRequestParameters();
 
-		$footerModel = new DesignSystemGlobalFooterModel( $params[ 'wikiId' ], $params[ 'lang' ] );
-
-		$this->setResponseData( $footerModel->getData() );
+		$this->setResponseData(
+			( new DesignSystemGlobalFooterModel( $params[ 'wikiId' ], $params[ 'lang' ] ) )->getData() );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wikiId" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)